app/views/questions/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <p id="notice"> <%= notice %> </p>
 
-<body class="question_show">
+<body class="container">
```

Comment:
Every single one of the `body` tags.

Reply:
Will not do this again. Thanks for letting me know.